app/services/invoices/calculate_fees_service.rb

```diff
@@ -161,6 +169,21 @@ def charge_included_in_next_subscription?(charge, subscription)
       next_subscription_charges.pluck(:billable_metric_id).include?(charge.billable_metric_id)
     end
 
+    def should_create_minimum_commitment_true_up_fee?(invoice_subscription)
+      subscription = invoice_subscription.subscription
+
+      return false if subscription.plan.pay_in_advance?
+      return false unless should_create_yearly_subscription_fee?(subscription)
+
+      calculate_true_up_fee_result = Commitments::Minimum::CalculateTrueUpFeeService.call(invoice_subscription:)
+
+      return false if calculate_true_up_fee_result.amount_cents == 0
```

Comment:
Style/NumericPredicate: Use `calculate_true_up_fee_result.amount_cents.zero?` instead of `calculate_true_up_fee_result.amount_cents == 0`. (https://rubystyle.guide#predicate-methods)